var/spack/repos/builtin/packages/python/package.py

```diff
@@ -945,3 +945,23 @@ def remove_files_from_view(self, view, merge_map):
                 view.remove_file(src, dst)
             else:
                 os.remove(dst)
+
+    def test(self):
+        self.run_test('true', expected=['not in output'])
+
+        # contains python executable
+        python = which('python')
+        assert os.path.dirname(python.path) == os.path.dirname(
+            self.command.path)
```

Comment:
This test will fail for `python@3:~pythoncmd`. The executable may not be called `python`. Also, the `import_tests` function is probably a better test anyway. Maybe combine the two.